pandas/core/window/common.py

```diff
@@ -70,6 +70,7 @@ def _apply(
         floor: int = 1,
         is_weighted: bool = False,
         name: Optional[str] = None,
+        use_numba_cache: Optional[bool] = False,
```

Comment:
can you add a doc-string here

Reply:
Shouldn't the type of ``use_numba_cache`` be just ``bool``, instead of ``Optional[bool]``?

Reply:
Yes good catch.